media-autobuild_suite.bat

```diff
@@ -1355,10 +1355,10 @@ if not exist "%instdir%\%msys2%\msys2_shell.cmd" (
         echo.- Downloading and unpacking msys2 basic system
         echo.
         echo -------------------------------------------------------------------------------
-        7z >nul 2>&1 && set mabs7z=7z || set mabs7z=7za
+        7z >nul 2>&1 || (
         7za >nul 2>&1 || powershell -NoProfile -NonInteractive -Command ^
-                ^(New-Object System.Net.WebClient^).DownloadFile^('https://github.com/chocolatey/chocolatey.org/raw/master/chocolatey/Website/7za.exe', '7za.exe'^)
-        %mabs7z% x msys2-base.tar.xz -so | %mabs7z% x -aoa -si -ttar -o..
+                ^(New-Object System.Net.WebClient^).DownloadFile^('https://github.com/chocolatey/chocolatey.org/raw/master/chocolatey/Website/7za.exe', '7za.exe'^))
```

Comment:
```suggestion
                ^(New-Object System.Net.WebClient^).DownloadFile^('https://github.com/chocolatey/chocolatey.org/raw/master/chocolatey/Website/7za.exe', '7za.exe'^)
        )
```
For the bat file, since there are no auto-formatters and require formatting by hand and confirmation so it would be much better to close the parenthesis on the same column as the opening command for readability